### PR TITLE
Project with specific transformation

### PR DESCRIPTION
**Pull request by nCastle1**
This sample demonstrates how to use the GeometryEngine with a specified transformation to project a geometry from one coordinate system to another. 

There is no visual map component to this sample, just a display of the before and after coordinates of the geometry.

### CODE REVIEW

**Comment by nCastle1** [comment]
@ThadT I initially requested your review, but I want to just confirm that this is the right approach with the sample designer first. Please feel free to disregard.

**Comment by nCastle1** [comment]
@ThadT I've confirmed with the sample designer that this is ready for review. Can you please take a look?

File: src/Android/Xamarin.Android/Samples/GeometryEngine/ProjectWithSpecificTransformation/ProjectWithSpecificTransformation.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+using Android.Widget;
+using Esri.ArcGISRuntime.Geometry;
+
+namespace ArcGISRuntimeXamarin.Samples.ProjectWithSpecificTransformation
+{
+    [Activity]
+    public class ProjectWithSpecificTransformation : Activity
+    {
+        // Label for showing the coordinates before projection
+        private TextView _lblBefore;
```
Comment (ThadT) [inline src/Android/Xamarin.Android/Samples/GeometryEngine/ProjectWithSpecificTransformation/ProjectWithSpecificTransformation.cs]: Not digging the Hungarian notation ;)

File: src/Android/Xamarin.Android/Samples/GeometryEngine/ProjectWithSpecificTransformation/ProjectWithSpecificTransformation.cs [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+        private void Initialize()
+        {
+            // Create a point geometry in NYC in WGS84
+            MapPoint myPoint = new MapPoint(-73.984513, 40.748469, SpatialReferences.Wgs84);
+
+            // Update the UI with the initial coordinates
+            _lblBefore.Text = $"x: {myPoint.X}, y: {myPoint.Y}";
+
+            // Create a geographic transformation step for transfrom WKID 108055, WGS_1984_To_MSK_1942
```
Comment (ThadT) [inline src/Android/Xamarin.Android/Samples/GeometryEngine/ProjectWithSpecificTransformation/ProjectWithSpecificTransformation.cs]: "transfrom" => "transform" (needs to be fixed for all platforms)

File: src/Android/Xamarin.Android/Samples/GeometryEngine/ProjectWithSpecificTransformation/ProjectWithSpecificTransformation.cs [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            GeographicTransformationStep geoStep = new GeographicTransformationStep(108055);
+
+            // Create the transformation
+            GeographicTransformation geoTransform = new GeographicTransformation(geoStep);
+
+            // Project to a coordinate system used in New York, NAD_1983_HARN_StatePlane_New_York_Central_FIPS_3102
+            MapPoint myAfterPoint = (MapPoint)GeometryEngine.Project(myPoint, SpatialReference.Create(2829), geoTransform);
```
Comment (ThadT) [inline src/Android/Xamarin.Android/Samples/GeometryEngine/ProjectWithSpecificTransformation/ProjectWithSpecificTransformation.cs]: Don't like the "my" naming convention either (personal preference again, feel free to disregard ;)). Down the road, I think we might need to come to a consensus on some specific naming conventions.

File: src/Android/Xamarin.Android/Samples/GeometryEngine/ProjectWithSpecificTransformation/metadata.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+  "Name": "Project with specific transformation",
+  "SampleName": "ProjectWithSpecificTransformation",
+  "Description": "This sample demonstrates how to use the GeometryEngine with a specified geographic transformation to transform a geometry from one coordinate system to another. ",
+  "Instructions": "See https://developers.arcgis.com/net/latest/wpf/guide/coordinate-systems-and-transformations.htm for more information about geographic coordinate systems, geographic transformations, and projected coordinate systems. ",
```
Comment (ThadT) [inline src/Android/Xamarin.Android/Samples/GeometryEngine/ProjectWithSpecificTransformation/metadata.json]: Is it possible to have these URLs formatted as links when output to the ReadMe.md?

**Comment by nCastle1** [comment]
Thanks for the review! I've made all of your suggested changes, including adding a comparison to the projection without specific transformation.

**Pull request merged**